diff --git a/cmake/extensions.cmake b/cmake/extensions.cmake
index c094056370..05e6ff6c1b 100644
--- a/cmake/extensions.cmake
+++ b/cmake/extensions.cmake
@@ -1451,6 +1451,7 @@ endfunction()

 # 3.5. File system management
 function(check_if_directory_is_writeable dir ok)
+  print(dir)
   execute_process(
     COMMAND
     ${PYTHON_EXECUTABLE}
@@ -1458,6 +1459,7 @@ function(check_if_directory_is_writeable dir ok)
     ${dir}
     RESULT_VARIABLE ret
     )
+  print(ret)

   if("${ret}" STREQUAL "0")
     # The directory is write-able
 