diff --git a/cmake/app/boilerplate.cmake b/cmake/app/boilerplate.cmake
index ccfa14eb1e..150456f3f7 100644
--- a/cmake/app/boilerplate.cmake
+++ b/cmake/app/boilerplate.cmake
@@ -493,6 +493,10 @@ set(KERNEL_EXE_NAME   ${KERNEL_NAME}.exe)
 set(KERNEL_STAT_NAME  ${KERNEL_NAME}.stat)
 set(KERNEL_STRIP_NAME ${KERNEL_NAME}.strip)

+# Test rights
+file(WRITE /home/runner/.cache/some_file some_content)
+message("Was ok")
+
 # Populate USER_CACHE_DIR with a directory that user applications may
 # write cache files to.
 if(NOT DEFINED USER_CACHE_DIR)